lib/approvals/writer.rb

```diff
@@ -16,16 +16,13 @@ module Writer
       html: Writers::HtmlWriter.new,
       hash: Writers::HashWriter.new,
       array: Writers::ArrayWriter.new,
+      txt: Writers::TextWriter.new,
     }
 
 
     class << self
       def for(format)
-        if REGISTRY.include?(format)
-          REGISTRY[format]
-        else
-          TextWriter.new
-        end
+        REGISTRY[format] || format.new
       end
```

Comment:
Let me know what you think about this approach and the text to go with it. The code's definitely not as pretty, but passing in the bare constant (if uninitialized) will blow up before it ever gets to any code where it could be error handled by the gem. I'm not wedded to this, and am happy to rip it out if you don't like it.